fuel-tx/Cargo.toml

```
@@ -14,13 +14,16 @@ description = "FuelVM transaction."
 derivative = { version = "2.2.0", default-features = false, features = ["use_core"] }
 fuel-asm = { workspace = true, default-features = false }
 fuel-crypto = { workspace = true, default-features = false }
+fuel-derive = { path = "../fuel-derive" }
```

This should be a workspace dep
```suggestion
fuel-derive = { workspace = true }
```